### PR TITLE
fix: [CDS-47424]: UI fixes for A/O svc env 

### PR DESCRIPTION
**Pull request by vikrant-harness**
### Summary

- Inline Edit Flow for service environment infrastructure fixed 
- Environments Page empty identifiers fix
- Org Level pipeline template with account level stage template added fixed support
- Empty API call failing while moving from single to multi env fixed

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by vikrant-harness** [comment]
retrigger titlecheck

File: src/modules/70-pipeline/components/PipelineInputSetForm/StageInputSetForm.tsx [hunk 1]
```diff
@@ -1065,7 +1065,7 @@ export function StageInputSetFormInternal({
         deploymentStage={deploymentStage}
         deploymentStageTemplate={deploymentStageTemplate}
         allowableTypes={
-          scope === Scope.PROJECT
+          scope === Scope.PROJECT || CDS_OrgAccountLevelServiceEnvEnvGroup
```
Comment (AshwinBhatkal-Harness) [inline src/modules/70-pipeline/components/PipelineInputSetForm/StageInputSetForm.tsx:1068]: I think this condition can be removed. I don't see this being used in account/org level as well

Comment (AshwinBhatkal-Harness) [reply src/modules/70-pipeline/components/PipelineInputSetForm/StageInputSetForm.tsx:1068]: Just passing allowableTypes should work

Comment (vikrant-harness) [reply src/modules/70-pipeline/components/PipelineInputSetForm/StageInputSetForm.tsx:1068]: When we select Account level stage Template and use it in org level pipeline template this comes into picture

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**